tools/debian/rules

```diff
@@ -11,6 +11,9 @@ endif
 %:
 	dh $@ --with=systemd
 
+clean:
+	dh $@
+
```

Comment:
Well, not "wrong", but too strong/unwieldy now for building under Fedora. I filed https://github.com/cockpit-project/cockpituous/pull/74 .